cachito/workers/pkg_managers/gomod.py

```diff
@@ -260,6 +253,39 @@ def resolve_gomod(app_source_path, request, dep_replacements=None, git_dir_path=
         return {"module": module, "module_deps": module_level_deps, "packages": packages}
 
 
+def _parse_name_and_version(list_deps_line: str) -> Tuple[str, str]:
+    """Parse package name and version (if present) from one line of go list -deps output."""
+    # line example: pkg.io/foo/bar pkg.io/foo/var v1.0.0 => pkg.io/foo/bar v1.0.1
+    parts = list_deps_line.split(" ")
+
+    # As far as we know, the possible forms of the dependency line are as follows (see
+    #   `go help list` and https://golang.org/ref/mod#go-mod-file-replace):
```

Comment:
@chmeliik Thank you for the information! In go help list I found only info about graph traversal order.